node-operators/block-producers/onboarding/run-a-block-producer/block-producer-docker.md

```diff
@@ -24,13 +24,26 @@ A Docker image combines the binary corresponding to the latest stable release of
 The following command to pull the Docker image:
 
 ```bash
-docker pull moondancelabs/parachain-dancebox
+docker pull moondancelabs/tanssi
 ```
 
 The command will download and extract the image and show the status upon execution:
 
 --8<-- 'code/node-operators/block-producers/run-a-block-producer/block-producer-docker/terminal/pulling-docker-image.md'
 
+### Setup the Data Directory {: #setup-data-directory }
+
+Running a block producer requires syncing with three chains: the relay chain, the Tanssi chain, and the Appchain it has been assigned to.
+
+Run the following command to create the directory where your block producer will store the databases containing blocks and chain states:
+
+```bash
+mkdir /var/lib/dancebox
+```
+
+!!! note
+    The directory is a parameter in the Docker start-up command. If you decide to create the directory elsewhere, update the command accordingly.
+
```

Comment:
closest i could get to it, but this also needs to be added to the docker command:

```
-u $(id -u ${USER}):$(id -g ${USER}) \
```

on moonbeam we have it like:

```
docker run --network="host" -v "/var/lib/moonbeam-data:/data" \
-u $(id -u ${USER}):$(id -g ${USER}) \
moonbeamfoundation/moonbeam:v0.36.0 \
```